app/components/margin-link.html

```diff
@@ -0,0 +1,39 @@
+<div class="container-fluid" ng-style="{
```

Comment:
Ah, yes. I broke this convention on purpose. :-P The reason is that we currently have a lot of files named index.ts and template.html. That's a bit annoying because text editors usually only display the filename and switching to the right file becomes an O(n) operation. ;-)

Let's decide together @ öffïce.